messages/en.json

```diff
@@ -0,0 +1,13 @@
+{
+  "Index": {
+    "title": "GameMatrix",
+    "titleRowRating": "MOST RATING GAMES",
+    "titleRowNew": "POPULAR GAMES FOR THIS WEEK"
```

Comment:
It's better to write not uppercase in translations and make text element in JSX with "text-transform: uppercase"